Source/WebCore/platform/mediastream/gstreamer/RealtimeIncomingSourceGStreamer.cpp

```diff
@@ -42,29 +42,28 @@ RealtimeIncomingSourceGStreamer::RealtimeIncomingSourceGStreamer()
     gst_bin_add_many(GST_BIN_CAST(m_bin.get()), m_valve.get(), queue, m_tee.get(), nullptr);
 
     gst_element_link_many(m_valve.get(), queue, m_tee.get(), nullptr);
-    gst_element_sync_state_with_parent(m_valve.get());
-    gst_element_sync_state_with_parent(queue);
-    gst_element_sync_state_with_parent(m_tee.get());
```

Comment:
Why is this redundant?

Reply:
The whole bin's state is sync'ed with its parent in `GStreamerMediaEndpoint::addRemoteStream()`.